crates/teloxide-macros/src/command_attr.rs

```diff
@@ -104,7 +104,16 @@ impl CommandAttr {
             "rename" => Rename(value.expect_string()?),
             "parse_with" => ParseWith(ParserType::parse(value)?),
             "separator" => Separator(value.expect_string()?),
-            "hide" => Hide,
+            "hide" => {
```

Comment:
Could you maybe just add an `expect_nothing()` method?

Reply:
@TheAwiteb